packages/contracts-bedrock/.storage-layout

```diff
@@ -68,28 +68,39 @@
 | l2Sender             | address                                                    | 50   | 0      | 20    | src/L1/OptimismPortal.sol:OptimismPortal |
 | finalizedWithdrawals | mapping(bytes32 => bool)                                   | 51   | 0      | 32    | src/L1/OptimismPortal.sol:OptimismPortal |
 | provenWithdrawals    | mapping(bytes32 => struct OptimismPortal.ProvenWithdrawal) | 52   | 0      | 32    | src/L1/OptimismPortal.sol:OptimismPortal |
-| paused               | bool                                                       | 53   | 0      | 1     | src/L1/OptimismPortal.sol:OptimismPortal |
+| spacer_53_0_1        | bool                                                       | 53   | 0      | 1     | src/L1/OptimismPortal.sol:OptimismPortal |
 | l2Oracle             | contract L2OutputOracle                                    | 53   | 1      | 20    | src/L1/OptimismPortal.sol:OptimismPortal |
 | systemConfig         | contract SystemConfig                                      | 54   | 0      | 20    | src/L1/OptimismPortal.sol:OptimismPortal |
-| guardian             | address                                                    | 55   | 0      | 20    | src/L1/OptimismPortal.sol:OptimismPortal |
+| superchainConfig     | contract SuperchainConfig                                  | 55   | 0      | 20    | src/L1/OptimismPortal.sol:OptimismPortal |
```

Comment:
I believe this is a safe overwrite to make.

Reply:
I agree this is safe to overwrite this